src/apps/sources/redux/actions.ts

```diff
@@ -14,6 +14,14 @@ export const editSourceAction = createActionThunk(
     EDIT_SOURCE_ACTION,
     api.update);
 
+export const resetCreateSourceAction = createActionThunk(
+    RESET_CREATE_SOURCE_ACTION,
+    api.create);
+    
+export const resetEditSourceAction = createActionThunk(
+    RESET_EDIT_SOURCE_ACTION,
+    api.update);
```

Comment:
So, these implementations should be similar to [`resetConceptFormAction()`](https://github.com/openmrs/openmrs-ocl-client/blob/c9209d5c9e5e0b1a5202f8633d34d71a6921fc97/src/apps/concepts/redux/actions.ts#L229), e.g.

```ts
export const resetCreateSourceAction = () => {
  return (dispatch: Function) => dispatch(resetAction(CREATE_SOURCE_ACTION));
}
```